## Jumping” into gene regulation: Understanding the regulatory adaptation of transposable elements and their effect on gene expression in maize

Merritt Khaipho-Burch1, Taylor Ferebee2, Michelle Stitzer3, Edward S. Buckler1,2,3,4

Affiliations
1 Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY 14853
2 Department of Computational Biology, Cornell University, Ithaca, NY 14853
3 Institute for Genomic Diversity, Cornell University, Ithaca, NY 14853
4 USDA-ARS; Ithaca, NY, USA 14853

With the onset of climate change, developing crop varieties adapted to future environments is paramount in maintaining plant productivity. Breeding a crop like Zea mays (maize) for future environments requires that we understand how all 35,000 genes are regulated in the genome. In the 1950s, Barbara McClintock hypothesized that transposable elements - so-called “jumping genes” - could be responsible for controlling the individual activity of genes and could lead to the specialized evolution and adaptation of gene regulatory grammar in their host genomes. We now know that genes are heavily impacted by the nearly 350,000 transposable elements that comprise 85% of the maize genome sequence. We hypothesize that transposable elements have played a key role in adaptation to new environments and, on a smaller scale, provided an abundant source of regulatory information that can impact gene expression. To test this hypothesis, we used gene expression data from 7,500 maize varieties and 600 wild relatives related to maize. We built models that quantify how transposable elements impact gene regulation and adaptation to new environments. We identify numerous transposable element insertions regulating or disrupting gene expression, enriched transposable-element derived motifs in environmentally responsive genes, and describe other genomic features associated with these expression changes. These patterns can be used to guide plant breeders on how to design high-yielding maize varieties more suited to new environments and conditions.
