## Elucidating the extent of pleiotropy in maize and its functional relevance towards trait prediction

Merritt B. Khaipho-Burch1, Anju Giri2, Guillaume Ramstein3, Brandon Monier2, M. Cinta Romay2, Edward S. Buckler1,2,4

1 Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY 14853
2 Institute for Genomic Diversity, Cornell University, Ithaca, NY USA 14853
3 Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark, 8000 Aarhus
4 USDA-ARS; Ithaca, NY, USA 14853


Pleiotropy, due to its magnitude and prevalence in maize, has been shown to have widespread effects in traits such as flowering time, leaf architecture, and inflorescence morphology. However, the genome-wide impact that pleiotropy has across all maize phenotypes is largely unknown. This project investigates the extent to which pleiotropy impacts phenotypes within maize through GWAS summary statistics reanalyzed from previously published physiological, morphological, metabolic, and expression phenotypes across the Nested Association Mapping population (NAM) and Goodman Association Panel. Due to the high genetic heterogeneity within and between these populations, we use a combination of global and local principal components to control for linked loci arising from population structure and kinship in our association models. From our GWAS reanalysis of approximately 250,000 different traits in maize, we obtained over a billion SNP-trait relationships. We hypothesize that although pleiotropy is common across maize genomes, inferring true pleiotropy and casual relationships from statistical noise is confounded by linkage disequilibrium and other factors. Knowledge of how the pleiotropic nature of loci impacts traits has applications in plant breeding programs to further understand the genotype to phenotype landscape, uncover interactions between functional loci, and potentially improve trait prediction models.
