## Limited contribution of transposable elements to regulatory adaptation in maize inbreds and hybrids

Merritt Khaipho-Burch1*, Michelle Stitzer2, Terry Casstevens2, Zack Miller2, M. Cinta Romay2, Edward S. Buckler1,2,3

Affiliations
1 Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY 14853
2 Department of Computational Biology, Cornell University, Ithaca, NY 14853
3 Institute for Genomic Diversity, Cornell University, Ithaca, NY 14853
4 USDA-ARS; Ithaca, NY, USA 14853

Transposable elements (TEs) contribute to ~85% of the maize genome and have been shown to impact genome structure and gene regulation by rearranging, adding, disrupting, or silencing endogenous promoters. In maize, large-effect TE insertions have implications on phenotypes, such as flowering time and the shift to apical dominance. The effect that TEs have on the expression of non-domestication genes during normal plant development is less understood and typically confounded by trans-regulatory sequences, which are unlinked from the focal gene and act through diffusible products. Because cis-regulatory sequences are linked to the gene they impact, we partition the variance in gene expression into the highly heritable cis-only component, allele-specific expression, and measured expression to determine which TEs cause substantial effects on gene expression. Using RNAseq data generated in inbred and hybrid genotypes created from the founders of the maize Nested Association Mapping panel, we identified less than 1% of all transposable element insertions regulating or disrupting the expression of roughly 14% of maize genes across non-stressed tissues. We then characterized the types of genes impacted through gene ontology enrichment and describe other genomic features associated with these expression changes. These patterns can be used to guide plant breeders on how to design high-yielding maize varieties more suited to new environments and conditions. 
