## Estimation of the divergence between maize and sorghum using the Andropogoneae

Merritt Burch1, Baoxing Song2, Cinta Romay2, Elizabeth Kellogg3, Edward Buckler2,4

1 Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY 14853
2 Institute for Genomic Diversity, Cornell University, Ithaca, NY USA 14853
3 Donald Danforth Plant Science Center, St. Louis, MO USA 63132
4 USDA-ARS; Ithaca, NY, USA 14853

The identification of regions showing evolutionary conservation between two species is limited by the breadth of experimental manipulation or surveys of existing variation between species. Such variation that exists between species is often confounded by deep coalescence events and divergent physiology. A more relevant sample of independent evolution spanning one billion years can be obtained by sequencing ~900 grasses in the Andropogoneae clade. All of these grasses exhibit NADP-ME C4 photosynthesis and are thought to share a common ancestor that existed between 16 and 20 million years ago. Here we outline our study using assembled Andropogoneae genomes to estimate evolutionary rates between maize and sorghum applying various methods to estimate base conservation. Genomic Evolutionary Rate Profiling (GERP) is one method of estimating constraint from sequence alignments. GERP estimates will be compared between three groups: solely within the Andropogoneae, within the Poaceae, and with available angiosperm genomes. By performing this analysis over broad evolutionary time scales, patterns of evolutionary constraint can be obtained. This information could help to infer a more accurate picture of the divergence between maize and sorghum and help to inform plant breeders on opportunities for heterosis effects and loci for gene editing. 