## A meta-GWAS reanalysis of twenty years of quantitative traits in maize


### Video presentation available here:
Poster video available on YouTube: [Merritt Burch - A meta GWAS reanalysis of twenty years of quantitative traits in maize](https://youtu.be/9lwoF-2EXDc)


Merritt B. Burch1, Anju Giri2, Guillaume Ramstein2, M. Cinta Romay2, Edward S. Buckler1,2,3

1 Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY 14853
2 Institute for Genomic Diversity, Cornell University, Ithaca, NY USA 14853
3 USDA-ARS; Ithaca, NY, USA 14853

Here we remap, via GWAS, phenotypes from across maize populations to find the features that characterize complex traits. Past mapping experiments have been difficult to compare to each other because they have focused on one population, utilize different genome versions, and have only analyzed a few traits. In this study, we collected thousands of previously published morphological, metabolic, and expression phenotypes across maize landraces (n = 5000), public inbred lines (n = 2815), and half-sib lines (n = 5000) in multiple environments. We then mapped this dataset in a meta-GWAS using consistent model parameters. Due to the high genetic heterogeneity within and between these populations, we use a combination of global and local principal components to control for linked loci arising from population structure and kinship. We hypothesize that the properties of these mapped variants can predict loci that have a functional impact, uncover regulatory networks of gene activity, reveal genotype by environment effects, and uncover loci showing statistical pleiotropy. We will compile these results in an online, queryable database for maize geneticists and breeders to mine for associations of interest for gene prioritization and gene editing.