## Regulatory adaptation of transposable elements and their effect within maize inbreds and hybrids

Merritt Khaipho-Burch1, Taylor Ferebee2, Michelle Stitzer3, Edward S. Buckler1,2,3,4

Affiliations
1 Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY 14853
2 Department of Computational Biology, Cornell University, Ithaca, NY 14853
3 Institute for Genomic Diversity, Cornell University, Ithaca, NY 14853
4 USDA-ARS; Ithaca, NY, USA 14853

Emails
mbb262@cornell.edu
tf259@cornell.edu
mcs368@cornell.edu
esb33@cornell.edu

Transposable elements (TEs) contribute to ~85% of the maize genome and have been shown to impact genome structure and gene regulation by rearranging, adding, or disrupting endogenous promoters. In maize, large effect TE insertions have implications on phenotypes such as flowering time and the shift to apical dominance. The effect that TEs have on the expression of non-domestication genes is less understood and typically confounded by trans-regulatory sequences, which are unlinked from the focal gene and act through diffusible products. Because cis-regulatory sequences are linked to the gene they impact, we partition the variance in gene expression into the highly heritable cis-only component and allele-specific expression rather than measured expression, to determine which TEs in cis cause substantial effects on gene expression. Using RNAseq data generated in inbred and hybrid genotypes created from the founders of the maize Nested Association Mapping panel, we are in the process of identifying numerous transposable element insertions regulating or disrupting gene expression, enriched transposable-element derived motifs in environmentally responsive genes, and describe other genomic features associated with these expression changes. These patterns can be used to guide plant breeders on how to design high-yielding maize varieties more suited to new environments and conditions. 